packages/runtime/datastore-definitions/api-report/datastore-definitions.api.md

```diff
@@ -112,7 +112,7 @@ export interface IFluidDataStoreRuntime extends IEventProvider<IFluidDataStoreRu
     // (undocumented)
     readonly IFluidHandleContext: IFluidHandleContext;
     // (undocumented)
-    readonly logger: ITelemetryLogger;
+    readonly logger: ITelemetryBaseLogger;
```

Comment:
This is a breaking change, we'll want a changeset with it. @markfields I think guidance to move away from `ITelemetryLogger.sendTelemetryEvent()/sendErrorEvent()/sendPerformanceEvent()` should be to now use `ITelemetryBaseLogger.send()` and add the category manually in order to log a telemetry/error/performance event?

Reply:
The guidance would be to immediately do `createChildLogger(logger)` which will give you an `ITelemetryLoggerExt` which you can use to log with the functions we all know and love.